sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineScaleSet.java

```diff
@@ -396,6 +396,13 @@ RunCommandResult runShellScriptInVMInstance(
      */
     Plan plan();
 
+    /**
+     * Get orchestration mode of the Virtual Machine Scale Set.
+     * Scale set orchestration modes allow you to have greater control over how virtual machine instances are managed by the scale set.
+     * @return orchestration mode of the virtual machine scale set
+     */
+    OrchestrationMode orchestrationMode();
```

Comment:
We only support getter for OrchestrationMode here, do we need to support withOrchestrationMode() interface as well?

Reply:
Yes, we are going to support creating with orchestrationMode feature. It needs a bit more attention and is not quite related to this issue, so I decided to open [another](https://github.com/Azure/azure-sdk-for-java/issues/25500) addressing this feature.